sycl/test-e2e/Matrix/joint_matrix_rowmajorA_rowmajorB.cpp

```diff
@@ -12,6 +12,8 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 // RUN: %if gpu %{ env IGC_JointMatrixLoadStoreOpt=2 %{run} %t.out %}
```

Comment:
Same here, if contiguous optimization does not get triggered for this test, we can remove `env IGC_JointMatrixLoadStoreOpt=2` option to save on run time.

Reply:
same here. it is to test accumulator 1x64. We can use 4-element vector loads, when flag is set to 2. If flag is set to 1, we use 1-element sub group loads.